pandatool/src/deploy-stub/deploy-stub.c

```diff
@@ -640,8 +694,21 @@ int main(int argc, char *argv[]) {
   ModuleDef *moddef;
   const char *log_filename;
   void *blob = NULL;
+#if defined(MS_WINDOWS)
+  char blob_path[MAX_PATH];
+#else
+  char blob_path[PATH_MAX];
+#endif
```

Comment:
We probably don't need to get the blob path unless we plan to load an external blob. Maybe just move this to map_external_blob() to match map_blob()?